lib/assets/javascripts/cartodb/table/menu_modules/filters/histogram.js

```diff
@@ -588,23 +588,22 @@
       return { min: minDate, max: maxDate}
 
     },  
-
+  
     abbreviateNumber: function (value) {
```

Comment:
@gfiorav I added some inline comments to help explain the function. Hope that helps.